demisto_sdk/commands/validate/sdk_validation_config.toml

```diff
@@ -7,7 +7,7 @@ ignorable_errors = [
     "MP106",
     "PA113", "PA116", "PA124", "PA125", "PA127", "PA129",
     "PB104", "PB105", "PB106", "PB107", "PB110", "PB111", "PB114", "PB115", "PB116", "PB118", "PB119", "PB121",
-    "RM100", "RM102", "RM104", "RM106", "RM108", "RM110", "RM112", "RM113",
+    "RM100", "RM102", "RM104", "RM106", "RM110", "RM112", "RM113",
```

Comment:
Why did you remove it?

Reply:
https://jira-dc.paloaltonetworks.com/browse/CIAC-10610, if I understand correctly, it needs to be removed to
fix all places in the content repo where the validation is ignored